woocommerce/woocommerce-android
# Dark Mode: Order Status Card

Partially implements #1587 by implementing light/dark material styling to the order status card. 

![Screen Shot 2020-02-07 at 5 12 27 PM](https://user-images.githubusercontent.com/5810477/74074814-1359d880-49cd-11ea-9e22-b58285ce828e.png)

![Screen Shot 2020-02-07 at 5 12 41 PM](https://user-images.githubusercontent.com/5810477/74074819-1654c900-49cd-11ea-977a-eb95219f721b.png)


## To Test
- Verify order status card styling in light theme on API 29
- Verify order status card styling in dark theme on API 29
- Verify order status card styling in light theme on API 23 
- Verify order status card styling in dark theme on API 23
